app/controllers/chat_channels_controller.rb

```diff
@@ -181,4 +182,12 @@ def render_chat_channel
       render json: { errors: @chat_channel.errors.full_messages }
     end
   end
+
+  def send_open_notification
+    if @chat_channel.channel_type != "direct"
```

Comment:
this check might be warranted to become a method in `.chat_channel`, something like:

```ruby
  def group?
    channel_type != "direct"
  end
```

what do you think?

Reply:
Is a good suggestion. Let me add this is new commit.